xtos/include/sof/lib/dma.h

```diff
@@ -243,6 +243,8 @@ struct dma_chan_data {
 	/* true if this DMA channel is the scheduling source */
 	bool is_scheduling_source;
 
```

Comment:
Add a comment here, like: 
`/* true if this DMA channel is registered, or already in use */`

Reply:
So is this a DMA channel tracking issue i.e. which DMA to free on stream stop ? Fwiw, not seeing this on timer domain with multiple DMA channels in operation.